changelog.json

```diff
@@ -1,5 +1,16 @@
 {
   "releases": {
+    "1.4.0-beta1": [
+      "[New] Release notes for update are now displayed within Desktop - #2774",
+      "[New] Detect merge conflicts when comparing branches - #4588",
+      "[Fixed] Avoid double checkout warning when opening a pull request in Desktop - #5375",
+      "[Fixed] Clear error in Publish repository dialog on change of tab - #5422. Thanks @Daniel-McCarthy!",
+      "[Fixed] Disable affected menu items when on detached HEAD - #5500. Thanks @say25!",
+      "[Fixed] Show border when commit description is expanded. - #5506. Thanks @aryyya!",
+      "[Fixed] Clone URL with matching GitHub repository would incorrectly choose GitHub repository - #4154",
```

Comment:
`Cloning gitlab URL results in GitHub repository being cloned`?

Reply:
or `Cloning GitLab URL can result in GitHub repository being cloned`?

Reply:
I think this would work with non-GitLab remotes that just happened to be formatted in the same `https://host.com/owner/name` - I wanted to focus on that behaviour